packages/preact-iso/router.d.ts

```diff
@@ -0,0 +1,19 @@
+import { VNode } from 'preact';
+
+export function LocationProvider(props: { children?: VNode }): VNode;
```

Comment:
Haven't tested, but maybe we can simplify that to

```tsx
export const LocationProvider: FunctionComponent;
```

At least the types in core seems to hint that `children` are typed by default: https://github.com/preactjs/preact/blob/b1a42a1bc61bf1f648e19a41cdf54d235810714c/src/index.d.ts#L86-L90